geoportal/c2cgeoportal_geoportal/views/entry.py

```diff
@@ -67,8 +66,8 @@
 from c2cgeoportal_geoportal.views.layers import get_layer_metadatas
 
 _ = TranslationStringFactory("c2cgeoportal")
-log = logging.getLogger(__name__)
-cache_region = get_region()
+LOG = logging.getLogger(__name__)
+CACHE_REGION = get_region()
```

Comment:
Using upper case for variables is not common, usually reserved for constants.

Reply:
For my IDE it's a constant and I agree with him...

Reply:
I was sure you was using vim, what is your IDE ?

Reply:
VisualStudio Code :-)